en/services/timesync_v1.md

```diff
@@ -0,0 +1,45 @@
+# Time Synchronization Protocol v1
+
+This protocol is used to estimate the time offsets between system clocks on two MAVLink systems, in a two-system MAVLink network.
+This is useful because it allows ....
+
+The protocol uses just one message [TIMESYNC](#TIMESYNC), which has two `int64_t` fields, `tc1` and `ts1`. 
+The message is sent out with a timestamp in `ts1` (current unix timestamp *1000).
+This is mirrored back by the remote system, along with its own timestamp.
+The first system can then determine the round-trip time, and estimate the timestamp offset.
+This sequence is run multiple times to remove the transient channel and processor usage from the offset calculation.
```

Comment:
Note that originally I thought this was because the timesync would drift with time. But actually it is because the round trip time, and time spent on each leg is not deterministic. So we need to average out the values and filter outliers.